src/Servers/Kestrel/Core/src/Internal/KestrelServerImpl.cs

```diff
@@ -207,7 +208,16 @@ async Task OnBind(ListenOptions options)
             }
             catch (Exception ex)
             {
-                Trace.LogCritical(0, ex, "Unable to start Kestrel.");
+                // Do not log stack trace for known errors #29801
```

Comment:
Code and issues have a bad habit of moving around.
```suggestion
                // Do not log stack trace for known errors https://github.com/dotnet/aspnetcore/issues/29801
```